tests/unit/pages/SplashPage.spec.js

```diff
@@ -42,6 +46,7 @@ jest.mock('src/composables/UserAuthentication', () => ({
   login: jest.fn(),
   getUserSessionToken: jest.fn(),
   initUserInformation: jest.fn(),
+  initUserRoles: jest.fn(),
```

Comment:
useless if you do not check that the function is correctly called. So add it !

Reply:
done